app/celery/service_callback_tasks.py

```diff
@@ -68,34 +94,34 @@ def _send_data_to_service_callback_api(self, data, service_callback_url, token,
         current_app.logger.info(
             "%s sending %s to %s, response %s",
             function_name,
-            notification_id,
+            object_id,
             service_callback_url,
             response.status_code,
         )
         response.raise_for_status()
     except RequestException as e:
         current_app.logger.warning(
-            "%s request failed for notification_id: %s and url: %s. exception: %s",
+            "%s request failed for id: %s and url: %s. exception: %s",
             function_name,
-            notification_id,
+            object_id,
             service_callback_url,
             e,
         )
         if not isinstance(e, HTTPError) or e.response.status_code >= 500 or e.response.status_code == 429:
             try:
-                self.retry(queue=QueueNames.CALLBACKS_RETRY)
-            except self.MaxRetriesExceededError:
+                self.retry(queue=retry_queue)
+            except self.MaxRetriesExceededError as e:
                 current_app.logger.warning(
```

Comment:
After this change we would get warnings not errors if the for `MaxRetriesExceededError`. That's probably fine, but just worth double checking. We don't consistently act on these logs, but I'm not sure if we consider Inbound SMS data "more important" than delivery status data since it does consist of actual messages and not just information about the messages.

Reply:
Good spot.

I'm really undecided but I decide to make them all error level. It might mean more sentry errors, but we don't have a good way of dealing with this for the moment so I'd rather we keep getting told about it and so incentivised to come up with a better solution than just ignore the fact callbacks might be broken for service teams...

I've pushed up a new commit